engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/model/element/ExecutableEndEvent.java

```diff
@@ -71,4 +82,8 @@ public boolean isTerminateEndEvent() {
   public void setTerminateEndEvent(final boolean isTerminateEndEvent) {
     this.isTerminateEndEvent = isTerminateEndEvent;
   }
+
+  public boolean isSignalEndEvent() {
+    return signal != null;
+  }
```

Comment:
🔧 We can move this method closer to the other `isXEndEvent()` methods. That keeps easy to read.